docs/src/rules/no-constant-binary-expression.md

```diff
@@ -53,6 +53,12 @@ const value4 = new Boolean(foo) === true;
 const objIsEmpty = someObj === {};
 
 const arrIsEmpty = someArr === [];
+
+const shortCircuit1 = condition1 && false && condition2;
+
+const shortCircuit2 = condition1 || true || condition2;
+
+const shortCircuit3 = condition1 && "non-nullish" && condition2;
```

Comment:
```suggestion
const shortCircuit3 = condition1 ?? "non-nullish" ?? condition2;
```

I believe this was intended to be `??`. Otherwise, the example would be valid for this rule.